docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: '3.9'
+
+services:
+  ln_giveaway_backend:
+    platform: linux/amd64
+    build: .
+    container_name: ln_giveaway_backend
+    command: ./entrypoint.sh
+    restart: on-failure
+    volumes:
+      - ./:/usr/src/app/
+    env_file:
+      - ./.env
+    depends_on:
+      - ln_giveaway_db
+    ports:
+      - 6000:6000
+    environment:
+      - CONSUMER_KEY=${CONSUMER_KEY}
+      - CONSUMER_SECRET=${CONSUMER_SECRET}
+      - ACCESS_KEY=${ACCESS_KEY}
+      - ACCESS_SECRET=${ACCESS_SECRET}
+      - LN_API_KEY=${LN_API_KEY}
+      - SQLALCHEMY_DATABASE_URI=postgresql://ln_giveaway:ln_giveaway@ln_giveaway_db:5432/ln_giveaway_dev
+      - SQLALCHEMY_TRACK_MODIFICATIONS=False
+
+  ln_giveaway_db:
```

let's rename this to just postgres.
```suggestion
  postgres:
```

---

I believe the database name can be whatever you want. Also, this is a dev config, not prod. Is there a major reason we should use postgres

---

Reason being, we do not want to give service names the same as the project. the name should be scoped to the operation, in this case postgres. AFAIK, this is best practice

---

Does this mean we have to stick to postgres on prod. Also, I have lots of postgre containers running on my local. With a name like `ln_giveaway_db` it easier to know what/whose container I'm working on or need to act on.

---

in that case lets use `ln_giveaway_pg_db`